src/routing/core.js

```diff
@@ -33,7 +33,8 @@ const coreBuildPaths = [
   "/dengue",
   "/ebola",
   "/enterovirus",
-  "/flu",
```

Comment:
Note that a request to `/flu` will now 404. Perhaps we should add a redirect here to send it to `/seasonal-flu` and then let the manifest-derived redirects take over.

Reply:
Do you mean to add another redirect in `src/redirects.js` that redirects `/flu` to `/seasona-flu`?

Reply:
Yeah. Or maybe to `/pathogens`. Thoughts?

Reply:
Redirecting to `/seasonal-flu` will match the current routing behavior, so maybe we use that? Seems fine to me either way.

Reply:
Updated PR so that `/flu` goes to `/seasonal-flu` (and if a version descriptor's attached it is preserved)